spec/functions/number.md

```diff
@@ -413,6 +413,107 @@ contains an implementation-defined currency value
 of the _operand_ of the annotated _expression_,
 together with the resolved options' values.
 
+#### The `:percent` function
+
+> [!IMPORTANT]
+> The _function_ `:percent` has a status of **Draft**.
+> It is proposed for inclusion in a future release of this specification and is not Stable.
+
+The function `:percent` is a selector and formatter for percent values.
+
+##### Operands
+
+The function `:percent` requires a [Number Operand](#number-operands) as its _operand_.
+
+When either selecting or formatting the _expression_,
+the _resolved value_ of the _operand_ is multiplied by 100.
+
+##### Options
+
+Some options do not have default values defined in this specification.
+The defaults for these options are implementation-dependent.
+In general, the default values for such options depend on the locale,
+the value of other options, or both.
+
+> [!NOTE]
+> The names of _options_ and their _option values_ were derived from the
+> [options](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#options)
+> in JavaScript's `Intl.NumberFormat`.
+
+The following _options_ are REQUIRED to be available on the function `:percent`:
+
+- `signDisplay`
+  - `auto` (default)
+  - `always`
+  - `exceptZero`
+  - `negative`
+  - `never`
+- `useGrouping`
+  - `auto` (default)
+  - `always`
+  - `never`
+  - `min2`
+- `minimumFractionDigits`
+  - _digit size option_, default: `0`
+- `maximumFractionDigits`
+  - _digit size option_, default: `0`
+- `minimumSignificantDigits`
+  - _digit size option_
+- `maximumSignificantDigits`
+  - _digit size option_
+- `trailingZeroDisplay`
+  - `auto` (default)
+  - `stripIfInteger`
+- `roundingPriority`
+  - `auto` (default)
+  - `morePrecision`
+  - `lessPrecision`
+- `roundingMode`
+  - `ceil`
+  - `floor`
+  - `expand`
+  - `trunc`
+  - `halfCeil`
+  - `halfFloor`
+  - `halfExpand` (default)
+  - `halfTrunc`
+  - `halfEven`
+
+When formatting or selecting, each of the options is applied
+after the _resolved value_ of the _operand_ is multiplied by 100
```

Comment:
This seems misleading? The options apply to the formatting (or selection). They don't alter the value itself.

The example that follows illustrates the application of options to the formatting, but isn't really an example of what this text currently says. 

Perhaps:
```suggestion
The _resolved value_ of the _operand_ is multiplied by 100
prior to formatting or selection.
Each _option_ is applied to that value.
```